src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -212,7 +216,7 @@ const useCloudEnvironmentPolling = googleProject => {
   }
   const load = async maybeStale => {
     try {
-      const [newDisks, newRuntimes] = googleProject ? await Promise.all([
+      const [newDisks, newRuntimes] = !!googleProject ? await Promise.all([
```

Comment:
In my judgment, considering empty to be falsy is a bug. In this case, a bug in the language spec. My preference would be:
```suggestion
      const [newDisks, newRuntimes] = !_.isEmpty(googleProject) ? await Promise.all([
```